docs/source/nlp/prompt_learning.rst

```diff
@@ -338,17 +335,19 @@ The inference file can contain a mix of prompts from all the tasks the model has
 .. code::
 
     python megatron_gpt_eval.py \
-            virtual_prompt_model=True \
-            model_file=PATH_TO_MODEL \
+            virtual_prompt_model_file=PATH_TO_NEMO_PROMPT_LEARNING_MODEL_FILE \
+            model_file=PATH_TO_FROZEN_GPT_MODEL_FILE \
```

Comment:
Maybe this variable name should be more descriptive - got_model_filepath, or base_modek_filepath